frontend/Styleguide/TextArea.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, fireEvent } from "@testing-library/react";
```

Comment:
Just curious when one would choose fireEvent over userEvent.

Reply:
user-event is an optional extension to testing library that I've never needed before. `fireEvent()` doesn't perfectly simulate every aspect of user events, but it's close enough for every situation I've ever encountered. I generally use fireEvent since it works for me and doesn't require another dependency to be added to the project. Am not opposed to user-event, either, though